tests/e2e/pubsub_grpc/pubsub_grpc_test.go

```diff
@@ -5,7 +5,7 @@
 // Licensed under the MIT License.
 // ------------------------------------------------------------
 
-package pubsubapp_e2e
```

Comment:
can we keep the e2e in the namespace somehow?

Reply:
I think that would not be possible if we want to write [Effective Go](https://golang.org/doc/effective_go), where underscores, and mixedCaps are not recommended. 

I took a look at other projects to see how they solve this naming problem and found that they are not including `e2e` in package name 
Example 
- nomad  https://github.com/hashicorp/nomad/blob/main/e2e/connect/connect.go
- kubernetes https://github.com/kubernetes/kubernetes/blob/master/test/e2e/auth/certificates.go

Reply:
Consider that we do not call those packages outside e2e, remove suffix is safe and make it clear.